src/pages/Controllers/_controllers.scss

```diff
@@ -1,9 +1,93 @@
+@import "vanilla-framework/scss/vanilla";
+
 .controllers {
+  th {
+    padding-bottom: 6px;
+    padding-top: 6px;
+  }
+
   td,
   th {
     &:first-child {
       min-width: 25%;
       white-space: nowrap;
     }
   }
+
+  &--count {
+    font-size: 0.875rem;
+    padding: 0.25rem 1.5rem;
+  }
+
+  &--header {
+    border-bottom: 1px solid $color-dark;
+    color: $color-mid-dark;
+    display: inline-grid;
+    font-size: 0.76562rem;
+    font-weight: 400;
+    grid-template-columns: repeat(2, 50%);
+    margin-bottom: 1rem;
+    padding: 6px 0 4px 16px;
+    text-transform: uppercase;
+    width: 100%;
+
+    .controllers__heading {
+      padding-top: 0.8rem;
+    }
+
+    .controllers--register {
+      text-align: right;
+    }
+
+    button {
+      margin: 0;
+      max-width: 15rem;
+    }
+  }
+
+  .register-a-controller {
+    margin-top: 1rem;
+    text-align: right;
+  }
+
+  .slide-panel__content {
+    width: 550px;
```

Comment:
Can you change this to `max-width` (for small screens)

Reply:
done